x-pack/plugins/secops/public/components/timeline/body/column_headers/headers.ts

```diff
@@ -7,10 +7,10 @@
 import { ColumnHeader } from './column_header';
 
 /** The default minimum width of a column */
-export const DEFAULT_COLUMN_MIN_WIDTH = 100;
+export const DEFAULT_COLUMN_MIN_WIDTH = 95;
 
 /** The default minimum width of the Event column */
-export const DEFAULT_EVENT_COLUMN_WIDTH = 300;
+export const DEFAULT_EVENT_COLUMN_WIDTH = 40;
```

Comment:
Please restore this min width (and the others like the one above it) to the original value, as this is not a usable width:

```
It would
cause
the log
message
to wrap
like this`

Reply:
I was optimizing for the small space but as discussed this should be restored for now and I should begin work on spacing and resizing with the flyout